### DISCUSSION TITLE
Reminding someone to review your pull request

### DISCUSSION THREAD

**Discussion by ttogola** [community/community#21550] [New to GitHub]
<p>After making changes to a pull request, what is the GitHub etiquette for notifying someone to take a look at the PR again? Should I write a comment <span class="mention">@mentioning</span> them? Should I do nothing and wait until a maintainer notices and decides to review/merge? Is there another way?</p>
<p>Thanks</p>

**Comment by winstliu** [comment, marked as answer]
<p>Anyone who previously reviewed your PR or is otherwise following the conversation will get a notification whenever you push new commits.  So what I would do is wait for a bit and see if the reviewer responds to the notification, and if not, then politely <span class="mention">@mention</span> them again saying that you’ve implemented the requested changes.  Hope that helps :slight_smile:</p>
